cmake/SSGCommon.cmake

```diff
@@ -226,6 +226,23 @@ if (SSG_SHELLCHECK_BASH_FIXES_VALIDATION_ENABLED AND SHELLCHECK_EXECUTABLE AND "
     endif()
 endmacro()
 
+
+set(PYTEST_COVERAGE_OPTIONS)
```

Comment:
what is this for? do you mean `unset(PYTEST_COVERAGE_OPTIONS)`?

Reply:
why is this variable guarded by ENABLE_PYTHON_COVERAGE but not the other stuff? If I set it to disabled it will all fail, right?